background_scripts/background_script.js

```diff
@@ -0,0 +1,27 @@
+/*
+  A user pointed out that the storage.sync feature actually doesn't work at all.
+  This background script attempts to copy any old data stored in `storage.sync` 
+  over to `storage.local`, which will be used from now on.
+*/
+
+function onError(error) {
+  console.log(error);
+}
+
+function mergeSyncData(localData) {
```

Comment:
The method is name implies that a merge is done. But instead data is just copied from sync to local. I suggest renaming the method or doing a merge of local and storage.